### ISSUE TITLE
Export Dbeaver connections to an external file

### ISSUE THREAD

**Issue by sachinwalia2k8** [serge-rider/dbeaver#633]
Is it possible to export all the connections from Dbeaver to an external file and import that in a different instance of dbeaver? If not then it may be a nice feature for people working with different machines as well as sharing connection info among teams.

**Comment by serge-rider** [comment]
DBeaver connections are stored in file `.dbeaver-data-sources.xml` (in DBeaver workspace, which is in %{HOME}/.dbeaver by default). You can copy it as is or just a part of it (it is XML so it won't be hard).

**Comment by nagromc** [comment]
Please note that, from now on, `.dbeaver-data-sources.xml` seems to be located in `%{HOME/.dbeaver4/General/.dbeaver-data-sources.xml`.

From DBeaver v4 I guess.

**Comment by jozzhart** [comment]
Recent update

https://github.com/dbeaver/dbeaver/wiki/Admin-Manage-Connections

**Comment by tcarrio** [comment]
For users who are getting here looking for where this is stored on a Linux distribution, I found the following while poking around.

First off, I had no folder in my home directory for DBeaver. `$HOME/.dbeaver` did not exist. So if it exists for you, that's great news, it's probably right there.

I did end up finding a folder at `$HOME/.local/share/DBeaverData`. This is the default `XDG_DATA_HOME` folder, but I didn't find it documented in the Wiki above. The structure otherwise seems similar, but nested under a workspace folder.

In the end, I found the config in `$HOME/.local/share/DBeaverData/workspace6/General/.dbeaver/data-sources.json`. Hope this helps anyone else trying to find their config.
